orttraining/orttraining/python/training/_utils.py

```diff
@@ -43,6 +42,43 @@ def get_device_index_from_input(input):
         device_index = get_device_index(input.device)
     return device_index
 
+def get_device(module):
+    '''Returns the first device found in the `module`'s parameters
+
+    TODO: This approach assumes a single device per model
+    '''
+    device = torch.device('cpu')
```

Comment:
Just to check, does this mean that all modules without parameters (e.g. a `ORTModule(torch.nn.ReLU())`) will run on the CPU? Would it make sense to consider the device of the input tensor(s) as well if there are no params?

Reply:
I will double check what PyTorch would do in such case